ts/industries.ts

```diff
@@ -179,6 +179,7 @@ export const industryNames: Record<IndustryName, string> = {
 type FourStrings = [string, string, string, string];
 const [input2, input3, input4] = ['Unused Input Slot 2', 'Unused Input Slot 3', 'Unused Input Slot 4'];
 const [output2, output3, output4] = ['Unused Output Slot 2', 'Unused Output Slot 3', 'Unused Output Slot 4'];
+const Cordwood: FourStrings = ['Beams, Cordwood, Lumber, Logs', output2, output3, output4];
```

Comment:
I think you probably wanted:
```ts
const Cordwood: FourStrings = ['Beams', 'Cordwood', 'Lumber', 'Logs'];
```

Since this is only used in one place, you can put the strings inline below.


```suggestion
```

Reply:
well there is only one input but 4 different materials can be put in

Reply:
I see what you mean now. I would suggest using `['Logs', output2, output3, output4]` to match what the game shows.